Mentor Expectations.md

```diff
@@ -1,3 +1,12 @@
 # Mentor Expectations
-
-_Editor's Note: The leads have been directed by the team to draft a document outlining expectations for Mentors of the team. Those expectations will be added to this document once approved_
+ 
+ * Must follow all SPS codes of conduct regarding volunteers
+ * Should be interviewed
```

Comment:
```suggestion
```
```suggestion
```
I think that since this one is only appli when the mentor is first joining the team, it would be an expectation of that process

Reply:
5/21 Leads Meeting: Leads okay removing this